aviatrix-controller-build/variables.tf

```diff
@@ -0,0 +1,138 @@
+variable "mp_listings" {
+  type = map(string)
+
+  default = {
+    "PAID" = "ocid1.appcataloglisting.oc1..aaaaaaaawdptz5o42tuhg62defjimphzuck4fbrz4cilez2mle3bcs5t22wa"
+    "BYOL" = "ocid1.appcataloglisting.oc1..aaaaaaaavalzcaxbs4hwta4uefnkdh7fjgd3uuw7lwlk6pn2uvzglrn3zt2a"
+  }
+}
+
+variable "mp_byol_versions" {
+  type = map(string)
+
+  default = {
+    "6.3.0" = "6.3.1"
+  }
+}
+
+variable "mp_paid_versions" {
+  type = map(string)
+
+  default = {
+    "6.3.0" = "6.3"
+  }
+}
+
+variable "mp_byol_listing_resource_id" {
+  type = map(string)
+
+  default = {
+    "6.3.1" = "ocid1.image.oc1..aaaaaaaawyzirrayoeo5ivzkf57texkiu44ytnfifkmzry2xm5uu4otfk7cq"
+  }
+}
+
+variable "mp_paid_listing_resource_id" {
+  type = map(string)
+
+  default = {
+    "6.3" = "ocid1.image.oc1..aaaaaaaams7arhiyikfhpvo2kj35ky76csllxdnzijqw6ves4a5saaj63mva"
+  }
+}
+
+variable "product_version" {
+  default = "6.3.0"
+}
+
+variable "enabled" {
+  type = bool
+  default = true
+}
+
```

Comment:
Do we still need to keep this?

Reply:
should we use a new variable with number type to directly set the count value?

Reply:
> Do we still need to keep this?

should we use a new variable with number type to directly set the count value?